global/lang/en_GB/navbar.json

```diff
@@ -14,7 +14,7 @@
     "settings.global.theme.dark": "dark theme",
     "settings.global.theme.ultraDark": "ultra dark theme",
     "settings.global.theme.coldDark": "cold dark theme",
-    "settings.global.theme.system": "system theme",
+    "settings.global.theme.system": "website theme",

```

Comment:
no, system theme gets the theme of the operating system the user's on and adjusts to that